lib/config/configuration.go

```diff
@@ -654,59 +656,83 @@ func applyAuthOrProxyAddress(fc *FileConfig, cfg *servicecfg.Config) error {
 func applyLogConfig(loggerConfig Log, cfg *servicecfg.Config) error {
 	logger := log.StandardLogger()
 
+	var w io.Writer
 	switch loggerConfig.Output {
 	case "":
-		break // not set
+		w = os.Stderr
```

Comment:
Seems like this changes the behavior, is this what we actually want? It's not clear to me what was happening before these changes, were we simply not logging anything?

Reply:
I think prior to this it would have been using a default output of stderr.